Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionMatchPatternCocoa.mm

```diff
@@ -396,6 +396,33 @@
     return false;
 }
 
+HashSet<String> toStrings(const MatchPatternSet& matchPatterns)
+{
+    HashSet<String> stringsToReturn;
```

Comment:
Add this below: `setringsToReturn.reserveInitialCapacity(matchPatterns.size())` with a blank line after.